### PR TITLE
Add containerdisk annotations test

### PR DESCRIPTION
**Pull request by 0xFelix**
**What this PR does / why we need it**:

This adds a test for the appropriate templates which ensures that the
actual templates in a live cluster contain the containerdisk
annotations.

**Special notes for your reviewer**:

Merge after #418

**Release note**:

```release-note
None
```

### CODE REVIEW

File: automation/test_containerdisk_annotations.sh [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+set -ex
+shopt -s extglob
+
+# Any template can be used here as they all have the same version label, fedora is an arbitrary one
+ver_label=$( \
+  yq -o props '.metadata.labels | with_entries(select(.key == "template.kubevirt.io/version"))' \
+  dist/templates/fedora-server-large.yaml \
+)
+
+templates=(dist/templates/+(centos7|centos-stream|fedora|rhel8|rhel9)-!(saphana)-*.yaml)
```
Comment (ksimon1) [inline automation/test_containerdisk_annotations.sh:28]: what about others templates (ubutu, opensuse)?

Comment (0xFelix) [reply automation/test_containerdisk_annotations.sh:28]: There are no containerdisks for those. (currently)

**Pull request merged**